p4-16/psa/PSA.mdk

```diff
@@ -727,27 +727,24 @@ follow.
 
     if (ostd.clone) {
         if (ostd.clone_session_id value is supported) {
-	    cos = class_of_service configured for ostd.clone_session_id in PRE
-	    set((egress_port[0], instance[0]), .., (egress_port[n], instance[n])) =
-	        set of egress_port and instance pair in the
-	    	configured sessions for ostd.clone_session_id in PRE
-            trunc = truncate configured for ostd.clone_session in PRE
-            plen = packet_length_bytes configured for ostd.clone_session in PRE
+            from the values configured for ostd.clone_session_id in PRE {
+                cos = class_of_service
+                set((egress_port[0], instance[0]), ..., (egress_port[n], instance[n])) =
+                    set of egress_port and instance pairs
+                trunc = truncate
+                plen = packet_length_bytes
+            }
```

Comment:
No semantic changes here -- just trying to eliminate the duplication of the phrase "configured for ostd.clone_session in PRE"